crates/egui/src/ui_builder.rs

```diff
@@ -34,9 +34,20 @@ impl UiBuilder {
     ///
     /// You should give each [`Ui`] an `id_source` that is unique
     /// within the parent, or give it none at all.
+    #[deprecated = "Renamed id_salt"]
     #[inline]
-    pub fn id_source(mut self, id_source: impl Hash) -> Self {
-        self.id_source = Some(Id::new(id_source));
+    pub fn id_source(self, id_salt: impl Hash) -> Self {
+        self.id_salt(id_salt)
+    }
```

Comment:
You can remove this one actually, since `UiBuilder` is a new API that hasn't been released yet